slurm.spec

```diff
@@ -429,8 +429,6 @@ install -D -m755 etc/slurm.epilog.clean ${RPM_BUILD_ROOT}%{_sysconfdir}/slurm.ep
 install -D -m755 contribs/sjstat ${RPM_BUILD_ROOT}%{_bindir}/sjstat
 
 # Delete unpackaged files:
-rm -f $RPM_BUILD_ROOT/%{_libdir}/libpmi.a
```

Comment:
@lipari , you'll likely want to remove the .la files here, not necessarily the .a libs.